### ISSUE TITLE
Hide grade menu when a course is not graded

### ISSUE THREAD

**Issue by sgendry** [dmuneras/moodle-theme_archaius#31]
I don't know if this is an issue with this theme or a tweak that needs to happen somewhere inside of moodle. 

I have test courses that are not graded and don't want to show the "grade" option in the top menu, but have not yet found out how to do that.

1) In course settings / appearance / it is possible to hide the gradebook from student. Did that. No change.

2) Just to be thorough I then went to site administration / users / permissions / define roles and disabled everything to do with grades from the student profile. No luck. "Grade" still shows in the menu.

Your thoughts?

**Comment by dmuneras** [comment]
I added the gradebook to the custom menu by default. If you want to remove it, you have to edit a file (sorry)

![screen shot 2015-02-01 at 4 36 57 pm](https://cloud.githubusercontent.com/assets/344518/5993957/9777cb14-aa30-11e4-98f1-b0f668a66c9f.png)

Just remove this part:


        $course_id = $this->page->course->id;
        if (isloggedin() && $course_id > 1) {
            $branchlabel = get_string('grades');
            $branchurl   = new moodle_url('/grade/report/index.php?id='.$this->page->course->id);
            $branchtitle = $branchlabel;
            $branchsort  = 9000;
            $branch = $menu->add($branchlabel, $branchurl, $branchtitle, $branchsort);
        }

**Comment by sgendry** [comment]
Thank you Daniel. This is very helpful. (Editing files is not an issue.)
The only downside of this fix is that removing that code deletes the grade menu option from all courses, forever.

I certainly don't want to be greedy. See the following as just a display of curiosity: Is there a way to make the display of that grade menu conditional? E.g. tie that to the "moodle / course settings / appearance / hide the gradebook from student" option?

**Comment by dmuneras** [comment]
Yes, I think it has to be easy, I just have to take a look of the code.
